row_073_file_route.py

```
@@ -0,0 +1,41 @@
+
+@router.post("/upload-file",name="uploadfile")
+def uploadFile(file: UploadFile = File(...)):
+    try:
+        UPLOAD_DIRECTORY = "./uploads/" # Ensure the directory exists 
+        os.makedirs(UPLOAD_DIRECTORY, exist_ok=True)
+        file_location = os.path.join(UPLOAD_DIRECTORY, file.filename)
+        with open(file_location, "wb+") as file_object:
+            file_object.write(file.file.read())
+        file_url = f"http://localhost:8000/uploads/{file.filename}"
+
+        return {"info": f"file '{file.filename}' saved at '{file_location}' url '{file_url}'"}
```

Ruff (0.13.2)

Consider moving this statement to an `else` block

(TRY300)